controllers/quickstarts.go

```diff
@@ -65,12 +66,12 @@ func (r *StorageSystemReconciler) ensureQuickStarts(logger logr.Logger) error {
 	return nil
 }
 
-func (r *StorageSystemReconciler) deleteQuickStarts(logger logr.Logger) {
+func (r *StorageSystemReconciler) deleteQuickStarts(logger logr.Logger,instance *odfv1alpha1.StorageSystem) {
```

Comment:
There should be a space after `,`

Reply:
strange, My locally installed golint extension also missed this. Anyways updated the same